Integrations/Whois/Whois.py

```diff
@@ -8246,11 +8294,14 @@ def whois_command():
         ec['Emails'] = whois_result.get('emails')
         md['Emails'] = whois_result.get('emails')
 
+    ec['QueryStatus'] = 'Success'
+    md['QueryStatus'] = 'Success'
+
     context = ({
-        'Domain': {
-            'Name': domain
+        outputPaths['domain']: {
+            'Name': domain,
+            'Whois': ec
         },
-        'Domain.Whois(val.Name && val.Name == obj.Name)': ec
```

Comment:
We may still need this (or some form of it) to update existing entries in the context. Though I'm not entirely sure why it wasn't doing so, to begin with.

Reply:
it's there, note that 
outputPaths['domain'] = 'Domain(val.Name && val.Name == obj.Name)'

or do you mean we should have several entries for whois under domain?

Reply:
No, that's what I was looking for, just didn't see it. Looks good to me :)